larky/src/main/resources/vgs/nts.star

```diff
@@ -73,7 +79,12 @@ def render(
     return input
 
 
+def supports_dcvv(input, pan):
+    return vault.getValue(jsonpath_ng.parse(pan)).startswith(4)

```

Comment:
we should not use vault here, we must pass a preprocessed value here

Reply:
how do we work around this slava? without this we'd need to put the logic into the main script which is less ideal (we could if we must but the encapsulation will look wrong). could we pass a method reference?

e.g.

```python
nts.supports_dcvv(payload, selector, resolver=lambda: vault.get)
```